File: papi/lista2/exercicio9.py
print("Aqui pediremos uma temperatura em Farenheit e informaremos a mesma temperatura em Celsius.")

while True:
    try:
        temperatura_farenheit = float(input("Por favor a temperatura em Farenheit - use o ponto como separador decimal: "))
        
        break
    except ValueError:
        print("Por favor informe uma temperatura válida.")

print("O equivalente em Celsius:", 5 * (temperatura_farenheit - 32) / 9)

File: papi/lista2/exercicio9.kt
package lista2

fun main() {
    var temperaturaFarenheit: Double
    while (true) {
        try {
            print("Digite a temperatura em Farenheit, usando o ponto como separador decimal: ")
            val temperaturaFarenheitString = readLine() 
            temperaturaFarenheit = temperaturaFarenheitString!!.toDouble()
            break
        } catch (e: Exception) {
            println("Por favor, insira um valor válido para a temperatura em Farenheit, usando o ponto como separador decimal")
        }
    }
    val temperaturaCelsius = 5 * (temperaturaFarenheit - 32) / 9
    println("A temperatura equivalente em Celsius é $temperaturaCelsius")
}   